docs/metrics/index.md

```diff
@@ -14,6 +14,8 @@ biases, and its intended uses.
 
 <div class="grid cards" markdown>
 
+## :kolena-metrics-classification-icon-16: Classification Metrics
```

Comment:
Should we move the title out of the context of `<div class="grid cards" markdown>`?

<img width="949" alt="image" src="https://github.com/kolenaIO/kolena/assets/17771952/f80948ac-c132-4fa9-a8fe-efd0cb547381">

Reply:
Ah, I was unaware since it only shows one column on my screen. Should be fixed now!